p74/src/bin/rustcrypto_cmac_ctr.rs

```diff
@@ -1 +1,77 @@
-fn main() {}
+use aes::{
+    cipher::{KeyIvInit, StreamCipher},
+    Aes128,
+};
+use cmac::{Cmac, Mac};
+use std::io::{Read, Write};
+
+type Aes128Ctr64LE = ctr::Ctr64LE<Aes128>;
+
+fn cmac_tag(key: &[u8], data: &[u8]) -> Vec<u8> {
```

Comment:
You don't need to return `Vec` here, instead you should return a generic array (either `GenericArray<u8, U16>`, or `Output<Cmac<Aes128>>`).